README.md

```diff
@@ -195,12 +195,29 @@ Notes
 bundle exec ruby lib/count-users-by-year-for-deletion.rb
 ```
 
-#### Create groups.json
+#### Create groups (json and text)
 
+json
 ```
 curl $ZENDESK_URL/groups.json -v -u "$ZENDESK_USER_EMAIL/token:$ZENDESK_TOKEN" > data/groups.json
 ```
 
+text
+```
+bundle exec ruby lib/get-groups-list-to-file.rb
+```
+
```

Comment:
These lines are for creating groups file and the script `get-groups-list-to-file.rb` is not present in this pull request but it is present in another pull request (https://github.com/alphagov/zendesk-scripts/pull/50). Can we exclude them?

Reply:
Accepted and changed to suit.